contingency/contingency-dsl/src/test/java/com/powsybl/contingency/dsl/ProbabilityContingencyExtension.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.powsybl.contingency.dsl;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.contingency.Contingency;
+
+/**
+ * @author Paul Bui-Quang <paul.buiquang at rte-france.com>
+ */
+public class ProbabilityContingencyExtension implements Extension<Contingency> {
+    private Contingency contingency;
+
+    private Double probabilityBase = 1.0;
```

Comment:
Same remark about double

Reply:
i want it to be nullable, anyway, this is a test and not a real extension

Reply:
You have to choose: if it's nullable, you have to be consistent with the getter. If not, you have to change the variable type.

Even we are in unit test, it's not a good reason to not pay attention the way we are coding.